src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -44,9 +38,11 @@
 
     private transient final KubernetesCloud cloud;
 
-    @DataBoundConstructor
+    @Deprecated
```

Comment:
I think there is no need to deprecate it
It should call the other constructor `this(...)` to avoid duplication